js/render-thumbnails.js

```diff
@@ -1,6 +1,6 @@
 import {openBigPicture} from './gallery.js';
 
-function renderThumbnails(data) {
+function renderThumbnails (data) {
   const thumbnailsContainerElement = document.querySelector('.pictures');
   const thumbnailTemplate = document.querySelector('#picture').content.querySelector('.picture');
   const tempContainer = document.createDocumentFragment();
```

Comment:
Я бы хотел видеть так
```suggestion
  const thumbnailsContainerElement = document.querySelector('.pictures');
  const thumbnailTemplateElement = document.querySelector('#picture').content.querySelector('.picture');
  const fragment = document.createDocumentFragment();
```

Reply:
Исправил, спасибо